tests/test_feature_indexing.py

```diff
@@ -317,12 +317,24 @@ def test_open(*args, **kwargs):
         finally:
             shutil.rmtree(tmp_dir)
 
+    def test_read_back_index(self):
+        """Ensure that index files written with write_fai() can be read back"""
+        import locale
+        old_locale = locale.getlocale(locale.LC_NUMERIC)
+        try:
+            locale.setlocale(locale.LC_NUMERIC, 'en_US.utf8')
```

Comment:
Thanks for writing such an explicit test. I had the feeling that I was playing with fire by using system encodings and locales, but never had any issues. Did you write this PR because you encountered an issue in practice?